content/en/docs/concepts/security/secrets-good-practices.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Good practices for Kubernetes Secrets
+description: >
+  Principles and practices for good Secret management for cluster administrators and application developers.
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+
+{{<glossary_definition prepend="In Kubernetes, a Secret is an object that"
+term_id="secret" length="all">}}
+
+The following good practices are intended for both cluster administrators and
+application developers. Use these guidelines to improve the security of your
+sensitive information in Secret objects, as well as to more effectively manage
+your Secrets.
+
+<!-- body -->
+
+## Cluster administrators
+
+### Configure encryption at rest
+
+By default, Secret objects are stored unencrypted in {{<glossary_tooltip
+term_id="etcd" text="etcd">}}. You should configure encryption of your Secret
+data in `etcd`. For instructions, refer to
+[Encrypt Secret data at Rest](/docs/tasks/administer-cluster/encrypt-data/).
+
+### Configure RBAC policies for Secrets
+
+When planning your {{<glossary_tooltip term_id="rbac" text="Role-based Access Control">}} [(RBAC)](/docs/reference/access-authn-authz/rbac/) policies,
+consider the following guidelines for `Secret` objects. You should also follow the other guidelines in
+[RBAC good practices](/docs/concepts/security/rbac-good-practices).
+
+- Restrict `watch` or `list` access to only the most privileged, system-level
+  components.
+- Only allow cluster admistrators to access `etcd`. This includes read-only access.
+
+A user who can create a Pod that uses a Secret can also see the value of that
+Secret. Even if cluster policies do not allow a user to read the Secret
+directly, the same user could have access to run a Pod that then exposes the
+Secret. You can detect or limit the impact caused by Secret data being exposed,
+either intentionally or unintentionally, by a user with this access. Some
+recommendations include:
+
+*  Use short-lived Secrets
+*  Implement audit rules that alert on specific events, such as concurrent
+   reading of multiple Secrets by a single user
+
+### Improve etcd management policies
+
+Consider wiping ot shredding the durable storage used by `etcd` once it is
```

Comment:
typo?
```suggestion
Consider wiping or shredding the durable storage used by `etcd` once it is
```

Reply:
I can't believe I did this, thank you Rey